partcad/src/partcad/wrappers/wrapper_render_threejs.py

```
@@ -0,0 +1,68 @@
+import os
+import sys
+import json
+
+sys.path.append(os.path.dirname(__file__))
+import wrapper_common
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+from utils_ocp import tessellate
+
+
+def process(path, request):
```

Ruff (0.8.2)

Missing return type annotation for public function `process`

(ANN201)